File: programming-concepts/logical-operators/python/and_or_not.py
# Raspberry Pi Foundation
# Developed to be used alongside Isaac Computer Science,
# part of the National Centre for Computing Education
# Usage licensed under CC BY-SA 4


def main():
    difficulty = "Hard"
    score = 55
    game_over = False

    if not game_over:
        if difficulty == "Hard" and score > 60:
            print("Wow, you are doing great!")
        elif (difficulty == "Normal" or difficulty == "Easy") and score > 40:
            print("Good stuff, keep it up!")
        else:
            print("Keep going!")

            
# This code will run if this file is executed directly
# (i.e. not called by another program)
if __name__ == '__main__':
    main()

File: programming-concepts/logical-operators/c-sharp/and_or_not.cs
/*
Raspberry Pi Foundation
Developed to be used alongside Isaac Computer Science, part of the National Centre for Computing Education
Usage licensed under CC BY-SA 4

Note: This file is designed to be copied out and compiled on your machine.
In order for it to compile properly you need to ensure that the project name is the same as the "namespace" in this file.
To run this file you need to:
1. Copy the contents
2. Paste them into the C# IDE of your choice (Visual Studio, for example)
3. Change the namespace to match your project (if necessary)
4. Compile the program
5. Run the program
*/

using System;

namespace IsaacCodeSamples
{

    class LogicalOperators
    {
        // The Main method is the entry point for all C# programs
        public static void Main() {
            string difficulty = "Hard";
            int score = 55;
            bool gameOver = false;

            if (!gameOver) {
                if (difficulty == "Hard" && score > 60) {
                    Console.WriteLine("Wow, you are doing great!");
                }
                else if ((difficulty == "Normal" || difficulty == "Easy") && score > 40) {
                    Console.WriteLine("Good stuff, keep it up!");
                }
                else {
                    Console.WriteLine("Keep going!");
                }
            }
        }
        

    }
}
